src/components/sprite-info/sprite-info.jsx

```diff
@@ -0,0 +1,113 @@
+const classNames = require('classnames');
+const React = require('react');
+
+const Box = require('../box/box.jsx');
+const styles = require('./sprite-info.css');
+
+const xIcon = require('./icon--x.svg');
+const yIcon = require('./icon--y.svg');
+const showIcon = require('./icon--show.svg');
+const hideIcon = require('./icon--hide.svg');
+const draggableIcon = require('./icon--draggable.svg');
+const notDraggableIcon = require('./icon--not-draggable.svg');
+
+const SpriteInfo = props => (
+    <Box
+        className={classNames({
+            [styles.spriteInfo]: true
+        })}
+    >
+        <div className={styles.row}>
+            <div className={styles.group}>
+                <span className={styles.inputLabel}>
+                    {props.name}
+                </span>
+                <input
+                    className={classNames(styles.inputForm, styles.inputFormSpriteName)}
+                    placeholder="Name"
+                    type="text"
+                />
```

Comment:
These inputs (and the styles for them) should eventually be refactored out into their own components.